packages/field-plugin/src/createFieldPlugin/createFieldPlugin.ts

```diff
@@ -41,9 +41,25 @@ export const createFieldPlugin: CreateFieldPlugin = (onUpdateState) => {
   const { uid } = params
 
   const postToContainer = (message: unknown) => {
-    // TODO specify https://app.storyblok.com/ in production mode, * in dev mode
-    const origin = '*'
-    window.parent.postMessage(message, origin)
+    try {
+      // TODO specify https://app.storyblok.com/ in production mode, * in dev mode
+      const origin = '*'
+      window.parent.postMessage(message, origin)
+    } catch (err) {
+      // eslint-disable-next-line functional/no-throw-statement
+      throw new Error(
```

Comment:
There are other reasons why the content in the `try`-clause could fail. Could we create a function `isClonable`?

```ts
const isClonable = (value: unknown): boolean => {
  try {
    void structuredClone(value)
  } catch(_) {
    return false
  }
  return true
}
```

Then only throw the error on this line if it's not clonable. If it's clonable, re-throw `err`.

Reply:
Yes, `core-js` is one of the biggest JavaScript libraries, so it's fine to use 

```
import 'core-js/actual/structured-clone';
```